source/dapsenv/__init__.py

```diff
@@ -58,26 +68,34 @@ def create_files():
     # directories
     if not os.path.exists(HOME_DIR):
         os.makedirs(HOME_DIR)
+        log.debug("'%s' created", HOME_DIR)
 
     if not os.path.exists(LOG_DIR):
         os.makedirs(LOG_DIR)
+        log.debug("'%s' created", LOG_DIR)
 
     if not os.path.exists(TMP_DIR):
         os.makedirs(TMP_DIR)
+        log.debug("'%s' created", TMP_DIR)
 
     if not os.path.exists(BUILDS_DIR):
         os.makedirs(BUILDS_DIR)
+        log.debug("'%s' created", BUILDS_DIR)
```

Comment:
The `if not...` phrases seems very repetitive to me. Maybe refactor the code (see the following lines). Additionally, use `%r` instead of `%s` in the `log.debug()` function, as `%r` creates the quotes automatically:

```python
def createdir(path):
   """ ... add some useful docstring here ..."""
   if not os.path.exists(path):
      os.makedirs(path)
      log.debug("%r created", path)
```

Use it like this:

```python
  for p in (HOME_DIR, LOG_DIR, TMP_DIR, BUILDS_DIR):
      createdirs(p)
```